Core/clim-core/panes.lisp

```diff
@@ -1810,7 +1809,7 @@ which changed during the current execution of CHANGING-SPACE-REQUIREMENTS.
 
 (defmethod compose-space ((bboard bboard-pane) &key width height)
   (declare (ignore width height))
-  (make-space-requirement :width 300 :height 300))
+  (make-space-requirement :width 100 :height 100))
```

Comment:
for consistency make it obey specified width/height with default values 100/100.

Reply:
Oh, of course. Thanks.